src/idl_gen_rust.cpp

```diff
@@ -382,8 +382,13 @@ class RustGenerator : public BaseGenerator {
       code_.Clear();
       code_ += "// " + std::string(FlatBuffersGeneratedWarning());
       code_ += "extern crate flatbuffers;";
+      code_ += "extern crate serde;";
       code_ += "use std::mem;";
       code_ += "use std::cmp::Ordering;";
+      if (parser_.opts.rust_serialize) {
+        code_ +=
+            "use self::serde::ser::{Serialize, Serializer, SerializeStruct};";
```

Comment:
line limit

Reply:
What is the line limit? In my editor the semicolon is on column 78 for this statement.